addons/account_peppol/models/account_move_send.py

```diff
@@ -240,4 +238,16 @@ def action_what_is_peppol_activate(self, moves):
             }
             return action
         else:
-            return moves.action_send_and_print()
+            # go back to previous (send and print) action
+            # to avoid doing participant SML lookup again, we don't go through action_send_and_print
+            return {
+                'name': _("Send"),
+                'type': 'ir.actions.act_window',
+                'view_mode': 'form',
+                'res_model': 'account.move.send.wizard' if len(moves) == 1 else 'account.move.send.batch.wizard',
+                'target': 'new',
```

Comment:
I was hoping to be able to repoen the same wizard (I mean the exact same record, giving `res_id: self.id`) but unfortunately this is called in an `api.model`. :( maybe could be changed in master...